### ISSUE TITLE
Password file in crawler job config is not honored

### ISSUE THREAD

**Issue by TheReal1604** [dadoonet/fscrawler#348]
The password field in the crawler config is not honored. This is my job config:

```
{
  "name" : "crawl_cifsshare",
  "fs" : {
    "url" : "/mnt/cifsshare",
    "update_rate" : "5m",
    "excludes" : [ "~*" ],
    "json_support" : false,
    "filename_as_id" : false,
    "add_filesize" : true,
    "remove_deleted" : true,
    "add_as_inner_object" : false,
    "store_source" : false,
    "index_content" : false,
    "attributes_support" : true,
    "raw_metadata" : true,
    "xml_support" : false,
    "index_folders" : true,
    "lang_detect" : false
  },
  "elasticsearch" : {
    "username" : "fscrawler",
    "password" : "YourCrawlerPassword",
    "nodes" : [ {
      "host" : "127.0.0.1",
      "port" : 9200,
      "scheme" : "HTTP"
    } ],
    "index" : "myindex",
    "type" : "doc",
    "bulk_size" : 100,
    "flush_interval" : "5s"
  },
  "rest" : {
    "scheme" : "HTTP",
    "host" : "127.0.0.1",
    "port" : 8080,
    "endpoint" : "fscrawler"
  }
}
```

My elasticsearch system is protected with x-pack. FSCrawler asks me every time, I start that job, for the elasticsearch password. The username field is working fine and is used from the config file.

**Comment by dadoonet** [comment]
Running 2.2? Duplicate of #329 which is fixed in 2.3.
Try the SNAPSHOT (link on the readme page)

**Comment by TheReal1604** [comment]
@dadoonet yeah, i am running 2.2.

Thanks!
